modules/vye/app/models/vye/user_info.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+module Vye; end
```

Comment:
I'm only commenting here because this is not seen anywhere else in the app. But typical convention is to wrap the entire class in the module like this:

```
module Vye
  class UserInfo
    ...
  end
end
```

It's not technically wrong to do it this way, I'd only be potentially concerned that our autoloader would be tripped up with classes and modules defined this way

Reply:
I appreciate your thoughts on the typical convention of wrapping the entire class in the module. You're absolutely correct that the standard practice is to encapsulate the class within the module, as you've illustrated.

However, in this instance, I opted for less indentation to aid in the readability and simplicity of the code. I believe that the additional indentation can sometimes add unnecessary complexity, particularly when we're dealing with larger code bases.

While I understand the potential concern about our autoloader being confused with classes and modules defined in this manner, we didn't encounter any issues during the linting process. This led me to believe that this approach was acceptable for our specific scenario.

That being said, I am more than open to adhering to the standard convention if it's deemed necessary or if it aligns better with our overall coding practices. As always, my ultimate goal is to ensure the maintainability and efficiency of our code base.

Reply:
@binq I agree with Trevor. I would keep the configuration consistent.

Reply:
done!